### ISSUE TITLE
Gradle 8.1.1 task declaration problems

### ISSUE THREAD

**Issue by marcingrzejszczak** [spring-cloud/spring-cloud-contract#1911]
(3.1.x contract version) after having fixed the deprecation issues I get this one

```groovy
* What went wrong:
A problem was found with the configuration of task ':copySnippets' (type 'Copy').
  - Gradle detected a problem with the following location: '/home/marcin/repo/spring-cloud-contract/samples/standalone/webclient/http-server/build/stubs/META-INF/com.example/http-server-webclient-gradle/0.0.1/mappings'.
    
    Reason: Task ':verifierStubsJar' uses this output of task ':copySnippets' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':copySnippets' as an input of ':verifierStubsJar'.
      2. Declare an explicit dependency on ':copySnippets' from ':verifierStubsJar' using Task#dependsOn.
      3. Declare an explicit dependency on ':copySnippets' from ':verifierStubsJar' using Task#mustRunAfter.
    
    Please refer to https://docs.gradle.org/8.1.1/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```

@shanman190 :grimacing:

**Comment by marcingrzejszczak** [comment]
Fixed by https://github.com/spring-cloud/spring-cloud-contract/commit/5c6eac2453ac1761c9f8e1a1ac1c41d6e55b9f8d

**Comment by shanman190** [comment]
This one is pretty easy, so I'll open a PR for the actual fix to 3.1.x sometime today.

**Comment by marcingrzejszczak** [comment]
Thank you!
